.gitignore

```diff
@@ -1,3 +1,4 @@
 main.py
 rewrite.py
 discordoauth2/__pycache__
+.idea/
```

Comment:
I don't believe `.idea/` needs to be here.

Reply:
its another IDE files -It's  intellij IDE.

Reply:
Yeah if you use any of the jetbraiins ides